ExportTable/ExportTable/Extension.cs

```diff
@@ -91,7 +91,7 @@ internal static IEnumerable<string> GetInlineTableData(object selected)
             {
                 return null;
             }
-            var rowCount = 0;
+            var rowCount = dataSet.Tables[0].Rows.Count;
             var lines = new string[rowCount];
             var rowNum = 0;
 
```

Comment:
Wouldn't this still enable the original issue where the export fails when `rowCount == 0`? I think maybe we could leave ln 94 as-is and just add a guard clause for `rowCount`, `if (rowCount < 1) { return Enumerable.Empty<string>(); }` or something similar